pkg/collector/corechecks/servicediscovery/language/language.go

```diff
@@ -146,12 +150,24 @@ func FindInArgs(args []string) Language {
 	if len(args) == 0 {
 		return ""
 	}
-	for i := 0; i < len(args); i++ {
-		procName := path.Base(args[i])
-		// if procName is a known language, return the pos and the language
-		if lang, ok := procToLanguage[procName]; ok {
-			return lang
-		}
+
+	langs := languagedetection.DetectLanguage([]languagemodels.Process{&procutil.Process{
+		// Pid doesn't matter since sysprobeConfig is nil
+		Pid:     0,
+		Cmdline: args,
+		Comm:    args[0],
```

Comment:
Added a ticket to investigate changing this to exe as a future improvement: https://datadoghq.atlassian.net/browse/USMON-1206